packages/react-table/src/components/TableComposable/Td.tsx

```diff
@@ -134,7 +134,8 @@ const TdBase: React.FunctionComponent<TdProps> = ({
           },
           column: {
             extraParams: {
-              onCollapse: expand?.onToggle
+              onCollapse: expand?.onToggle,
+              expandId: expand?.expandId
```

Comment:
Should the `const compoundParams...` code that follows the lines above include a similar line addition as this? Right now the "composable table compound expandable" example doesn't have any ID set on the toggles.

Reply:
Possibly - I will double check

Reply:
Actually - upon double checking, I did add an `expandId` to the `TdCompoundExpand` type :) so both expand and compoundexpand should be good here :)